src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Migrate/PackageSpecProjectJsonMigrationCandidate.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using NuGet.LibraryModel;
+using NuGet.RuntimeModel;
+
+namespace NuGet.PackageManagement.VisualStudio.Migrate
+{
+    internal class PackageSpecProjectJsonMigrationCandidate
+    {
+        /// <summary>
+        /// List of dependencies that apply to all frameworks.
+        /// <see cref="ProjectStyle.PackageReference"/> based projects must not use this list and instead use the one in
+        /// the <see cref="PackageSpec.TargetFrameworks"/> property which is a list of the <see cref="TargetFrameworkInformation"/> type.
+        /// </summary>
+        public IList<LibraryDependency> Dependencies { get; init; }
+        public ImmutableArray<LibraryDependency> TargetFrameworkDependencies { get; init; }
```

Comment:
Please use the per tfm dependencies, so have a TFI that's within a framework. 

This will allow the migrator to raise an error in case a project.json has multiple frameworks which is unsupported.

Reply:
I've unsplit the dependencies. Thanks!